src/lib/kafka/producer.js

```diff
@@ -0,0 +1,153 @@
+/*****
```

Comment:
Why we need producer? As far as I remember this service only consumes so we can remove it. Sorry for leaving the producer.js- there :(

Reply:
OK - I can remove this if you're sure it's not worth keeping :+1: